docs/getting_started/slurm.md

```diff
@@ -0,0 +1,122 @@
+---
+icon: material/server-network
+---
+
+# 🔌 **Running on a SLURM cluster**
+
+_This documentation is a work in progress._
+
+OmniGibson can be run on a SLURM cluster using the _enroot_ container software, which is a replacement
+for Docker that allows containers to be run as the current user rather than as root. _enroot_ needs
+to be installed on your SLURM cluster by an administrator.
+
+With enroot installed, you can follow the below steps to run OmniGibson on SLURM: 
+
+1. Download the dataset to a location that is accessible by cluster nodes. To do this, you can use
+the download_dataset.py script inside OmniGibson's scripts directory, and move it to the right spot
+later. **This step is already done for SVL and Viscam nodes**
+```{.shell .annotate}
+OMNIGIBSON_NO_OMNIVERSE=1 python scripts/download_dataset.py
+mv omnigibson/data /cvgl/group/Gibson/og-data-0-2-1
```

Comment:
is there a gh action that updates these periodically? or e.g.: once every time we have a new release?

Reply:
Unfortunately not right now & this is not easy to do so I'll keep doing it manually.